index.tsx

```diff
@@ -124,7 +124,20 @@ function App() {
         // two reverse proxies.
         ADAPTABLE_TRUST_PROXY_DEPTH: "2",
     };
-    const env = mergeEnvSimple(dbEnv, standardEnv, config.environment);
+
+    // Remove PORT from the container environment
+    const { PORT, ...env } = mergeEnvSimple(dbEnv, standardEnv, config.environment) || {};
+
+    let port = 80;
+    if (PORT) {
+        const msg = `Invalid value '${PORT}' for PORT in Runtime Environment`;
+        const m = /^\s*(\d+)\s*$/.exec(PORT);
+        if (!m) throw new Error(msg);
+        const p = parseInt(m[1], 10);
+        if (!Number.isNaN(p) || p < 1 || p > 65535) throw new Error(msg);
```

Comment:
I believe this should be Number.isNan(p)

Reply:
Sorry for the confusion, should this be `Number.isNaN(p)` instead of `!Number.isNaN(p)`